### PR TITLE
Fixed SoilLayerTrait import

### LINKED ISSUE #67: SoilLayerTrait values are being imported as 0's

### PR DESCRIPTION
**Pull request by MikeStower**
Resolves #67

### CODE REVIEW

File: Core/Application/CQRS/Soils/InsertSoilLayerTraitsCommand.cs [hunk 1]
```diff
@@ -45,10 +45,8 @@ private Unit Handler(InsertSoilLayerTraitsCommand request)
 
             // All the non-key properties of an entity
             var layer_props = _context.GetEntityProperties(typeof(SoilLayer));
-            IEntity layer = null;
-            Func<IEntity, bool> layer_matches = other =>
-                    layer_props.All(i => i.GetValue(layer)?.ToString() == i.GetValue(other)?.ToString());
 
+            IEntity layer = null;
```
Comment (jbrider) [inline Core/Application/CQRS/Soils/InsertSoilLayerTraitsCommand.cs:49]: Why not declare this as a SoilLayer?

Comment (MikeStower) [reply Core/Application/CQRS/Soils/InsertSoilLayerTraitsCommand.cs:49]: Convenience mostly, parts of that method were copy/pasted from other import functions. I could change it if necessary but I don't think it really makes a difference. 

Though looking at it again it doesn't need to be declared outside the loop any more. Originally it was part of a local function which required it to be instantiated, but that's no longer the case.

Comment (MikeStower) [reply Core/Application/CQRS/Soils/InsertSoilLayerTraitsCommand.cs]: Cleaned it up in 42594e8

**Review by jbrider — COMMENTED** [review]
I'm having trouble working out which variables are what.  t is an item in trait, then an item in soils, then there is an s.Trait and a t.Trait. Single letter variables may work in a single line Linq expression but they are awful as soon as their point of definition is more than a line away.

There should also be a comment to explain where the data is coming from - particularly why there is an [i+3] reference. Magic number references could be changed to a variable name to explain what they're used for - removing the need for an extra comment. If I'm maintaining this code at a later date, my only indication of the data source is a reference to a Table from somewhere.

**Review by jbrider — APPROVED** [review]
That looks good.

**Pull request merged**